pom.xml

```diff
@@ -33,11 +33,11 @@
 
     <!-- dependency versions -->
     <alpn.agent.version>2.0.10</alpn.agent.version>
-    <ant.version>1.10.13</ant.version>
+    <ant.version>1.10.14</ant.version>
     <apache.avro.version>1.11.2</apache.avro.version>
     <apache.httpclient.version>4.5.14</apache.httpclient.version>
     <apache.httpcore.version>4.4.16</apache.httpcore.version>
-    <asciidoctorj-diagram.version>2.2.9</asciidoctorj-diagram.version>
+    <asciidoctorj-diagram.version>2.2.11</asciidoctorj-diagram.version>
```

Comment:
Doublecheck the asciidoctor plugin, I seem to remember it getting bumped back because images were generated into the wrong place.

Reply:
Seems that 2.2.11 fixed the issue.